src/graphql/log/tests.rs

```diff
@@ -518,22 +519,23 @@ async fn load_oplog() {
 }
 
 #[tokio::test]
+#[allow(clippy::too_many_lines)]
 async fn load_connection_by_prefix_timestamp_key() {
     let schema = TestSchema::new();
     let store = schema.db.op_log_store().unwrap();
     let generator: OnceLock<Arc<SequenceGenerator>> = OnceLock::new();
-    let mut key_list: Vec<Vec<u8>> = Vec::new();
-
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src1", 1, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src1", 2, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "review", "src1", 2, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "review", "src1", 3, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src1", 3, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src1", 4, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src2", 5, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "piglet", "src2", 6, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "review", "src1", 4, &generator).await);
-    key_list.push(insert_oplog_raw_event(&store, "review", "src2", 5, &generator).await);
+    let key_list: Vec<Vec<u8>> = vec![
+        insert_oplog_raw_event(&store, "piglet", "src1", 1, &generator),
```

Comment:
I think we need to use the public names of the the private repositories in these lines. I Could you modify these please? Besides, I think this could even be done in a separate issue if there are a lot of lines to rename.

Reply:
I think it’s better to address them in a separate issue, but I have a question: there are quite a few semi-supervised engine names—most of them in identifiers, with a minority in string literals. Is renaming the identifiers required?
In other words, what's the scope of the renaming?

Reply:
I only meant the sensor and the manager modules only in the above comment, but if you want to extend the scope to the semi supervise engine, I agree that a separate issue will be nicer, and the identifiers I believe should be renamed along with the literals. And it would also require updating giganto-client. Let me create the issues in bothd repos.

Reply:
Understood. I will address the sensor and manager names in this PR and handle the semi-supervised engine names in the separate issues.